clients/product-settings/src/SettingsAdmin.ts

```diff
@@ -75,165 +72,132 @@ export interface SettingsAdmin {
    * @param namespace A program-supplied namespace that is used to organize settings and prevent name collisions.
    * @param name The name of the setting. Acceptable characters are alphanumeric and the period character.
    * @param applicationSpecific Specifies whether the setting is specific to the current application, or used by all applications.
-  // SWB
-   * @param projectId The wsgId of the Project, if the settings is specific to a project, otherwise undefined.
-  // SWB
-   * @param iModelId The wsgId of the iModel, if the setting is specific to an iModel, otherwise undefined. The projectId must be specified if iModelId is specified.
+   * @param iTwinId The id of the iTwin, if the settings is specific to an iTwin, otherwise undefined.

```

Comment:
Is this comment true of how the settings are being used?

Reply:
Skip this one, I have a PR to completely remove PDSS client.